lib/mix_unused/analyzers/unreachable/usages.ex

```diff
@@ -45,12 +46,9 @@ defmodule MixUnused.Analyzers.Unreachable.Usages do
     Debug.debug(modules, &debug/1)
   end
 
-  @spec declared_usages([module() | mfa()], Exports.t()) :: [mfa()]
+  @spec declared_usages([Filter.mfa_pattern()], Exports.t()) :: [mfa()]
```

Comment:
Why not `Filter.pattern()`?

Reply:
Good find, that's an oversight, will fix it.